save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -593,22 +605,24 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
 
         div("row align-items-center justify-content-center") {
             nav("nav nav-tabs mb-4") {
-                ProjectMenuBar.values().forEachIndexed { i, projectMenu ->
-                    li("nav-item") {
-                        val classVal =
-                                if ((i == 0 && state.selectedMenu == null) || state.selectedMenu == projectMenu) " active font-weight-bold" else ""
-                        p("nav-link $classVal text-gray-800") {
-                            attrs.onClickFunction = {
-                                if (state.selectedMenu != projectMenu) {
-                                    setState {
-                                        selectedMenu = projectMenu
+                ProjectMenuBar.values()
+                    .filter { it != ProjectMenuBar.SETTINGS || state.selfRole.isHigherOrEqualThan(Role.ADMIN) }
```

Comment:
Btw why we have a special case here for SETTINGS tab? 
I think RUN tab is also should not be seen for those who cannot run the project (but this is discussable)

Reply:
In the case we want to disable `RUN` tab for viewers we also need to move project info and project description to another tab like it is in organizations because it might be good to let anyone see the description and so on

Reply:
Will continue this in #900